gradio/__init__.py

```diff
@@ -30,6 +34,7 @@
     Slider,
     StatusTracker,
     Textbox,
+    TimeSeries,
```

Comment:
why twuce, typo?

Reply:
Different spelling (or casing actually)

Reply:
TimeSeries vs Timeseries

Reply:
That is what you're referring to, right?